### ISSUE TITLE
Forbid incorrectly swapped variables

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#788]
# Rule request

## Thesis

We need to forbid code like:

```python
a = b
b = a
```

And:

```python
temp = a
a = b
b = temp
```

The first one indicates a failed attempt to swap variables.
The second one should be rewritten to: `a, b = b, a`

https://rust-lang.github.io/rust-clippy/master/index.html#almost_swapped

**Comment by milg15** [comment]
I might give it a try later

**Comment by sobolevn** [comment]
Please, feel free to send a PR in case you will have the time! 👍

**Comment by sobolevn** [comment]
Closed by https://github.com/wemake-services/wemake-python-styleguide/pull/877
